baklava-dry-run-1.env

```diff
@@ -37,6 +37,12 @@ OP_GETH__SYNCMODE=
 
 IS_CUSTOM_CHAIN=true
 
+# Path to the datadir, If the datadir is empty then a new datadir will be
+# initialised at the given path.
+#
+# The path can be absolute, or relative to the docker-compose.yml file.
+DATADIR_PATH=./envs/baklava-dry-run-1/datadir
```

Comment:
Should we default to `./envs/$NETWORK_NAME/datadir` or `./envs/$ENV_NAME/datadir` instead of requiring this?

Reply:
I don't think we can, I don't think it's possible to reference variables defined within the same env file.

Reply:
I think you can:
```
> cat .env
A=foo
B=${A}bar
> cat docker-compose.yml 
services:
  test:
    image: alpine
    command: ["echo", "$B"]
> docker compose config
name: d
services:
  test:
    command:
      - echo
      - foobar
    image: alpine
    networks:
      default: null
networks:
  default:
    name: d_default
```

Reply:
Oh, great!